java/private/create_jvm_test_suite.bzl

```
@@ -113,3 +117,34 @@ def create_jvm_test_suite(
         tags = ["manual"] + tags,
         visibility = visibility,
     )
+
+def _contains_label(haystack_str_list, needle):
+    absolute_needle = _absolutify(needle)
+    for haystack_str in haystack_str_list or []:
+        absolute_haystack_label = _absolutify(haystack_str)
+        if absolute_needle == absolute_haystack_label:
+            return True
+    return False
+
+def _absolutify(label_str):
+    repo = ""
+    package = ""
+    name = ""
+    if not label_str.startswith("@"):
+        repo = ""
+    else:
+        parts = label_str.split("//")
+        repo = parts[0]
```

this looks backwards and hard to read
```suggestion
     if label_str.startswith("@"):
        parts = label_str.split("//")
         repo = parts[0]
```
and rely on the initial setting of repo to blank to carry through.